math/benches/msm.rs

```diff
@@ -0,0 +1,124 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use lambdaworks_math::cyclic_group::IsCyclicBilinearGroup;
+use lambdaworks_math::elliptic_curve::traits::HasDistortionMap;
+use lambdaworks_math::msm::naive::Naive;
+use lambdaworks_math::{
+    elliptic_curve::{
+        curves::test_curve::{QuadraticNonResidue, ORDER_P},
+        element::EllipticCurveElement,
+        traits::HasEllipticCurveOperations,
+    },
+    field::{
+        element::FieldElement, fields::u64_prime_field::U64PrimeField,
+        quadratic_extension::QuadraticExtensionField,
+    },
+    msm::{pippenger, MSM},
+};
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
+
+type FE = FieldElement<U64PrimeField<ORDER_P>>;
+
+macro_rules! bench_msm_for_length {
```

Comment:
Using a macro here isn't worth it. A more readable way of doing this instead would be defining a well-named function that parametrizes the `msm_size`.

Reply:
Also, you should check this [benchmark setup](https://github.com/lambdaclass/AES_zero_knowledge_proof_circuit/tree/master/benches), it may be useful to see different approaches in how Criterion benchmarks could be structured.

Reply:
And lastly, all the comments that you made in the PR about this code are really good and could be very useful as comments in the code.

Reply:
@ilitteri, thanks for pointing that out. This was my first time trying the reporting feature from Criterion, and I think I got confused about requiring different function names for the grouping, but now I noticed that it wasn't needed.

I removed the macros and adopted a similar style as the link you suggested. It looks like a nice setup to include benchmarks for other modules.

Also, I added some extra comments in the benchmark code.

LMK if this looks good to you.

Reply:
Looks good, thanks!